master/.golangci.yml

```diff
@@ -84,79 +84,66 @@ linters-settings:
 linters:
```

Comment:
do you have a resolved list of which ones are effectively moving out of the disabled list through this PR?

Reply:
Yep should be 
```
        bidichk
        containedctx
        decorder
        durationcheck
        errname
        exhaustruct
        gofumpt
        gosimple
        grouper
        importas
        makezero
        nilerr
        noctx
        tenv
        tparallel
        usestdlibvars
```